examples/dbinfo.py

```diff
@@ -1,66 +1,12 @@
-from typing import Dict
+# This file containts information on to access the server
+
 from aperturedb import Connector
-import argparse
 
 DB_HOST = "localhost"
 DB_PORT = 55557
 DB_USER = "admin"
 DB_PASSWORD = "admin"
 
 
-def in_notebook():
-    try:
-        from IPython import get_ipython
-        if 'IPKernelApp' not in get_ipython().config:  # pragma: no cover
-            return False
-    except ImportError:
-        return False
-    except AttributeError:
-        return False
-    return True
-
-
-class DBInfo:
-    """
-    **Common helper methods needed for multiple example programs**
-    It is used in notebooks in the examples as well as the command line programs in there.
-    This file is kept at the top level, and symlinked to the specefic subfolder.
-    """
-
-    def __init__(self, mandatory_params: Dict = {}) -> None:
-        if in_notebook():
-            class Obj:
-                pass
-            self.params = Obj()
-            self.params.db_host = DB_HOST
-            self.params.db_port = DB_PORT
-            self.params.db_username = DB_USER
-            self.params.db_password = DB_PASSWORD
-        else:
-            self.params = self.get_args(mandatory_params)
-
-    def create_connector(self):
-        return Connector.Connector(
-            host=self.params.db_host,
-            port=self.params.db_port,
-            user=self.params.db_username,
-            password=self.params.db_password)
-
-    def get_args(self, mandatory_params: Dict):
-        parser = argparse.ArgumentParser()
-
-        # Database config
-        parser.add_argument('-db_host', type=str, default=DB_HOST)
```

Comment:
I sometimes run on a different port just to test things - by removing this it seems like there is no way to use prepare_aperturedb.py on a different port now?

Reply:
yea, that would be a useful option to specify. But that leads to some extra checks added (Things like checking for ipykernel, etc.) . I wonder if there could be an approach to handle this without adding additional logic into dbinfo.